cloudbuild.yaml

```diff
@@ -22,7 +22,7 @@ steps:
     env:
       - 'RUN_KUBE_TESTS=1'
     secretEnv: ['GITHUB_TOKEN']
-    args: ['-r']
+    args: ['-r', '-p', '-v']
```

Comment:
might be better not to do verbose in ci so we can more easily see the failures

Reply:
what I've been doing is writing to stdout anything that I wanted printed whether or not a test fails